### PR TITLE
add FSR instructions for Bottles

### PR DESCRIPTION
**Pull request by mirkobrombin**
We have often been asked how to activate the FSR in Bottles and I believe this is the place, at least for Pop! users.

### CODE REVIEW

File: content/use-amd-fsr.md [hunk 1]
```diff
@@ -78,3 +78,12 @@ Lutris version 0.5.9 includes a toggle for AMD FSR in each game's preferences. T
 * Enable the "Enable AMD FidelityFX Super Resolution (FSR)" toggle in the options below.
 
 Then follow the "Setup your In-game Resolution" steps above, and enjoy FSR via Lutris!
+
+### Bottles
+
+Bottles from version 2021.10.14 also includes a toggle for AMD FSR in the preferences of each bottle.
```
Comment (storyaddict) [inline content/use-amd-fsr.md:84]: This sentence is a bit unclear.

Are you saying: "WINE bottles created in Bottles version 2021.10.14 and later?

Comment (storyaddict) [reply content/use-amd-fsr.md]: Got it, thanks!

File: content/use-amd-fsr.md [hunk 2]
```diff
@@ -78,3 +78,14 @@ Lutris version 0.5.9 includes a toggle for AMD FSR in each game's preferences. T
 * Enable the "Enable AMD FidelityFX Super Resolution (FSR)" toggle in the options below.
 
 Then follow the "Setup your In-game Resolution" steps above, and enjoy FSR via Lutris!
+
+### Bottles
+
+For more information about the Bottles app refer to [this article](/articles/bottles.md)
```
Comment (Ampersandstorm) [inline content/use-amd-fsr.md:84]: This sentence is missing a period.

**Review by Ampersandstorm — APPROVED** [review]
Looks good.

**Review by ahoneybun — APPROVED** [review]
I just needed to remove the .md at the end of the link as it breaks the link. Otherwise looks good and thank you for your work!

**Pull request merged**